src/main/java/com/google/api/codegen/transformer/ruby/UnitTestCaseViewModel.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.ruby;
+
+import com.google.api.codegen.ruby.RubyGapicContext;
+import com.google.api.codegen.transformer.ModelTypeTable;
+import com.google.api.codegen.viewmodel.OptionalArrayMethodView;
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Method;
+import com.google.common.base.Preconditions;
+
+/** ViewModel of each unit test case for Ruby. */
```

Comment:
What is the reason to add another general UnitTestcaseViewModel given that we already have 'TestCaseView'? Is it because you want to use 'OptionalArrayMethodView'?

I discussed with @shinfan that we should include 'ApiMethodView' in the 'TestCaseView', so that's case, then that should address your need here.

Please discuss with @shinfan